webui/apps/docs/src/content/docs/docs/quickstart.md

```diff
@@ -12,106 +12,143 @@ description: Walk through prequisites for Mac and Linux
-"dev-mode".
+This command downloads dependencies required to run the agent machine and starts Eidolon in "dev-mode". Dev mode provides a local http server and local memory, making it easy to get comfortable with Eidolon functionality.
 
-🔎 The first time you run this command, you may be prompted to enter credentials that the machine needs to run 
-(ie, OpenAI API Key).
+>🔎 The first time you run this command, you will be prompted to enter the OpenAI API Key, and possibly other credentials. This information is stored in `eidolon-quickstart/.env`.
 
 If the server starts successfully, you should see the following output:
-```
-Starting Server...
-INFO:     Started server process [34623]
-INFO:     Waiting for application startup.
-INFO - Building machine 'local_dev'
+
+```text title=output
+ ✔ Network eidolon-quickstart_default           Created                  0.1s 
+ ✔ Container eidolon-quickstart-agent-server-1  Created                  0.1s 
+ ✔ Container eidolon-quickstart-webui-1         Created
+Attaching to agent-server-1, webui-1
+webui-1         |   ▲ Next.js 15.0.0-canary.123
+webui-1         |   - Local:        http://xxxx:3000
+webui-1         |   - Network:      http://xxx.xx.x.x:3000
+webui-1         | 
+webui-1         |  ✓ Starting...
+webui-1         |  ✓ Ready in 256ms
 ...
-INFO - Server Started in 1.50s
+agent-server-1  | INFO - Building machine 'local_dev'
+agent-server-1  | INFO - Starting agent 'hello-world'
+agent-server-1  | INFO - Collecting anonymous metrics, to disable metrics set DISABLE_ANONYMOUS_METRICS
+agent-server-1  | INFO - Server Started in 3.36s
 ```
+You are now up and running with a simple agent! 🎉
 
-You can also check out your machine's [swagger docs](http://localhost:8080/docs#/).
+🚨 Running into problems? Ask for help on <a href="https://discord.com/invite/6kVQrHpeqG" target=_blank>Discord</a>.
 
-Believe it or not, you are already up and running with a simple agent! 🎉
+## Try it out!
 
-🚨 Running into problems? Ask for help on [Discord](https://discord.com/invite/6kVQrHpeqG).
+Now that Eidolon is running, let's try it out! You can use the web UI or command line interface.
 
-## What just happened?
+### WebUI Developer Tool
 
-The repository you just cloned defines an **AgentMachine** 💻 with a single **AgentProgram** 🤖 named `hello-world` 👋.
+A _Developer Tool_ web application is deployed to your server on port 3000. You can use the Developer Tool with any of your Eidolon applications.
 
-The agent 🤖 is defined in a yaml file 📄 located at `resources/hello_world_agent.yaml`.
+To converse with your Eidolon agent for the first time:
 
-This file describes how to instantiate your agent from its **AgentTemplate** 🏭 and describes any customization you might
-want (like a custom LLM. tools, etc).
-
-```yaml
-apiVersion: eidolon/v1
-kind: Agent
-metadata:
-  name: hello-world
+1. Open a web browser to http://localhost:3000 (or replace localhost with your server URL).
```

Comment:
we can send them directly to the dev tool link to skip a step if you think that would be a better UX. I don't have an opinion here, just pointing out the possibility.

Reply:
Valid point. From a learning perspective, I think it's good to see this step.